modules/admin_manual/pages/configuration/server/security/oauth2.adoc

```diff
@@ -96,6 +96,19 @@ To install the application, place the content of the OAuth2 app inside your inst
 
 To enable token-only based app or client logins in `config/config.php` set `token_auth_enforced` to `true`.
 
+==== Trusting Clients
+
+Since version 0.5.0 of the OAuth2 app, you can mark clients as trusted. This will have the effect that the consent step in the authentication process will be skipped for this client.
+
+image:configuration/server/security/oauth2_configuration.png[image]
+
+If you want to mark an existing client as trusted, you have to copy the `Client Identifier (ID)` and the `Client Secret`, then delete the existing entry and finally add it again via the occ comand. Follow this link regarding xref:configuration/user/oidc/oidc.adoc[ownCloud client config details].
```

Comment:
Questions:
* **How** did these entries find their way into that screen?
* Copy some content as you can see in the screenshot - **ok**
* **Delete content** - from where, in the gui, per occ, what is the process in details, where to click or which command
*  Readding as trusted per command - **ok**

Reply:
> How did these entries find their way into that screen?

Desktop, Android and iOS are there per default, the owncloudweb entry was added by the admin. Is that relevant for this section of the documentation?

> Delete content - from where, in the gui, per occ, what is the process in details, where to click or which command

Good point. I just noticed a bug in the UI because there should be trashbin buttons on the right side but they are outside of the viewport. You can see them when you resize the webui to 90%.